bin/render-activity-stream-html.js

```diff
@@ -104,10 +104,9 @@ for (const src of ${JSON.stringify(scripts, null, 2)}) {
     <link rel="stylesheet" href="${options.baseUrl}css/activity-stream.css" />
   </head>
   <body class="activity-stream">
+    <div id="header-asrouter-container" role="presentation"></div>
     <div id="root">${isPrerendered ? html : "<!-- Regular React Rendering -->"}</div>
-    <div id="snippets-container">
-      <div id="snippets"></div>
-    </div>${options.noscripts ? "" : scriptTag}
+    <div id="footer-asrouter-container" role="presentation"></div>${options.noscripts ? "" : scriptTag}
```

Comment:
so looking back at this.... I am sure this breaks legacy snippets in 67 but they aren’t officially removed until 68. So we can either change our portal code to use `#snippets-container` or update the snippets code to use this or leave it broken. @piatra thoughts? Do we need legacy snippets in 67?

Reply:
this seems to work:

```
diff --git a/bin/render-activity-stream-html.js b/bin/render-activity-stream-html.js
index cf55ca6c..d76da095 100644
--- a/bin/render-activity-stream-html.js
+++ b/bin/render-activity-stream-html.js
@@ -106,7 +106,9 @@ for (const src of ${JSON.stringify(scripts, null, 2)}) {
   <body class="activity-stream">
     <div id="header-asrouter-container" role="presentation"></div>
     <div id="root">${isPrerendered ? html : "<!-- Regular React Rendering -->"}</div>
-    <div id="footer-asrouter-container" role="presentation"></div>${options.noscripts ? "" : scriptTag}
+    <div id="snippets-container" role="presentation">
+      <div id="snippets"></div>
+    </div>${options.noscripts ? "" : scriptTag}
   </body>
 </html>
 `;
diff --git a/content-src/asrouter/asrouter-content.jsx b/content-src/asrouter/asrouter-content.jsx
index 5acb4a62..c1c3b600 100644
--- a/content-src/asrouter/asrouter-content.jsx
+++ b/content-src/asrouter/asrouter-content.jsx
@@ -94,7 +94,7 @@ export class ASRouterUISurface extends React.PureComponent {
     this.state = {message: {}, bundle: {}};
     if (props.document) {
       this.headerPortal = props.document.getElementById("header-asrouter-container");
-      this.footerPortal = props.document.getElementById("footer-asrouter-container");
+      this.footerPortal = props.document.getElementById("snippets-container");
     }
   }
```